### PR TITLE
Updates to docs

### PR DESCRIPTION
**Pull request by tnscorcoran**
Added a skeleton Scenario 5

### CODE REVIEW

File: data/hackathon/scenario5.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Single Hybrid Cloud Platform for AI and Intelligent Apps
+exercise: 3
```
Comment (jmhbnz) [inline data/hackathon/scenario5.mdx:3]: Can you please update this to `exercise: 5`.

File: data/hackathon/scenario5.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+date: '2024-06-05'
```
Comment (jmhbnz) [inline data/hackathon/scenario5.mdx:4]: Update date to today.

**Review by jmhbnz — CHANGES_REQUESTED** [review]
Two tweaks before we merge.

File: data/hackathon/scenario5.mdx (hunk 1 above)
Comment (tnscorcoran) [reply data/hackathon/scenario5.mdx:3]: sure

**Review by jmhbnz — APPROVED** [review]
LGTM

**Pull request merged**